src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationAttribute.kt

```diff
@@ -255,7 +255,7 @@ enum class LocationUsage {
 }
 
 enum class AvailabilityStatus {
-  PROBATION_TEAM,
+  PROBATION_ROOM,
```

Comment:
A small change/rename on the back of a comment by @timharrison-moj on a previous PR.